src/Microsoft.ML.Data/Utilities/ComponentCreation.cs

```diff
@@ -126,7 +126,8 @@ public static IDataView LoadTransforms(this IHostEnvironment env, Stream modelSt
         /// <summary>
         /// Creates a data loader from the arguments object.
         /// </summary>
-        public static IDataLoader CreateLoader<TArgs>(this IHostEnvironment env, TArgs arguments, IMultiStreamSource files)
+        [BestFriend]
+        internal static IDataLoader CreateLoader<TArgs>(this IHostEnvironment env, TArgs arguments, IMultiStreamSource files)
```

Comment:
>CreateLoader [](start = 36, length = 12)

Thanks Ivan. I wouldn't necessarily do this now, but this entire assembly seems like a *lot* of stuff we do not want public. But not as part of this PR.

Reply:
Oh what I meant was the whole class. But like I said not part of this PR.